pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "build>=1.1.1", "wheel>=0.37"]
+requires = ["setuptools>=75.0", "build>=1.2.1", "wheel>=0.44"]
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.